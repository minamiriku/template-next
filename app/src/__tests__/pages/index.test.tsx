import React from 'react'
import HomePage from 'src/pages'
import { render } from '../utils'

test('ホームページ', () => {
  const { container } = render(<HomePage />)
  expect(container).toMatchSnapshot()
})
