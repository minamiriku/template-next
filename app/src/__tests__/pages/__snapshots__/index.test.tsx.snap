// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ホームページ 1`] = `
<div>
  <div
    class="css-0"
  >
    <h1
      class="css-0"
    >
      Hello Jest!
    </h1>
  </div>
</div>
`;
